### PR TITLE
Modified retrain to output graph with retrained node names prefixed with net-<network id>

### PR DESCRIPTION
**Pull request by deepak-matroid**
Changed retrain model name-space. serving & matroid repos need to be synced to interface correctly

### CODE REVIEW

File: tensorflow/examples/image_retraining/retrain.py [hunk 1]
```diff
@@ -164,6 +167,14 @@
 
 train_writer = None
 
+def get_network_prefix(network_id):
+  #Adds a prefix to variable names used for nodes trained for the speicific network
+  if network_id is not '':
+    return ('net-'+network_id + '/')
+  else:
+    return ''
+  pass
```
Comment (dong-matroid) [inline tensorflow/examples/image_retraining/retrain.py]: remove

File: tensorflow/examples/image_retraining/retrain.py [hunk 2]
```diff
@@ -225,32 +236,19 @@ def create_image_lists(image_dir, testing_percentage, validation_percentage):
...
+    number_validation = int(math.ceil(number_images*validation_percentage/100))
+    number_train = number_images - number_test - number_validation
+    print("Percentage Split Input(Val/Test)=%d/%d"%(validation_percentage,testing_percentage))
+    print("Number of Training/Validation/Test Images=%d/%d/%d"%(number_train, number_validation, number_test))
+    if(number_train==0 or number_validation==0 or number_test==0):
+      print("Atleast One Category has 0 images. Add more images or set correct train/test/val percentages")
+      return None
+    rand_permutation = np.random.permutation(number_images)
+    validation_images = [os.path.basename(file_name) for file_name in file_list[0:number_test]]
+    testing_images = [os.path.basename(file_name) for file_name in file_list[number_test:(number_test+number_validation)]]
```
Comment (dong-matroid) [inline tensorflow/examples/image_retraining/retrain.py]: actually use rand_permutation

File: tensorflow/examples/image_retraining/retrain.py (hunk 1 above)
Comment (deepak-matroid) [inline tensorflow/examples/image_retraining/retrain.py]: Should I be in-lining it then ?

**Comment by dong-matroid** [comment]
LGTM, thx @deepak-matroid

**Pull request merged**